src/Workspaces/CSharp/Portable/Extensions/SemanticModelExtensions.cs

```diff
@@ -451,9 +451,48 @@ public static ISet<INamespaceSymbol> GetUsingNamespacesInScope(this SemanticMode
                 }
             }
 
+            // for type constraints use declared accessibility of its member declaration (class, interface, delegate, method)
```

Comment:
Sentence case, and rationalize with the numbering in the rest of the comments in this method.